CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 - New config values `password_reset_token_lifetime`
   and `email_verification_token_lifetime`: https://github.com/supertokens/supertokens-core/issues/297
+- New JWT recipe to create JWT tokens using SuperTokens
```

Comment:
Update the changelog to specify db schema changes

Reply:
Done